### ISSUE TITLE
bug: passHost in ApisixUpstream is not recognized

### ISSUE THREAD

**Issue by sunkara007** [apache/apisix-ingress-controller#2010]
### Current Behavior

````
kubectl apply -f - <<EOF
apiVersion: apisix.apache.org/v2
kind: ApisixUpstream
metadata:
  name: springboot-template-upstream
spec:
  scheme: https
  passHost: node
  externalNodes:
    - type: Domain
      name: dev1.clouddev.net
      port: 443
EOF
````
Above upstream signature is giving error as `passHost` is unknown field

### Expected Behavior

Above yaml should be able to apply using ingress controller as per specification provided by https://github.com/apache/apisix-ingress-controller/blob/master/samples/deploy/crd/v1/ApisixUpstream.yaml

### Error Logs

strict decoding error: unknown field "spec.passHost"

### Steps to Reproduce

1. install APISIX along with ingress controller using helm chart
2. create ApisixUpstream as mentioned above 
3. check errors on console

### Environment

- your apisix-ingress-controller version (output of apisix-ingress-controller version --long): Image apache/apisix-ingress-controller:1.6.0

- your Kubernetes cluster version (output of kubectl version): Client Version: version.Info{Major:"1", Minor:"25", GitVersion:"v1.25.4", GitCommit:"872a965c6c6526caa949f0c6ac028ef7aff3fb78", GitTreeState:"clean", BuildDate:"2022-11-09T13:36:36Z", GoVersion:"go1.19.3", Compiler:"gc", Platform:"darwin/amd64"}
Kustomize Version: v4.5.7
Server Version: version.Info{Major:"1", Minor:"25+", GitVersion:"v1.25.12-eks-2d98532", GitCommit:"0aa16cf4fac4da27b9e9e9ba570b990867f6a3d8", GitTreeState:"clean", BuildDate:"2023-07-28T16:52:04Z", GoVersion:"go1.20.6", Compiler:"gc", Platform:"linux/amd64"}
- OS version if running APISIX Ingress controller in a bare-metal environment (run `uname -a`)

**Comment by mbuotidem** [comment]
I'm experiencing the same error.

**Comment by Revolyssup** [comment]
@sunkara007 This feature is available from 1.7.0 version of ingress controller.
